modules/representation_management/lib/representation_management/v0/pdf_constructor/form_2122a.rb

```diff
@@ -193,9 +193,10 @@ def appointment_options(data)
             # Consent Limits
             "#{PAGE2_KEY}.RelationshipToVeteran[1]": limitations_of_consent_text(data.consent_limits),
             # Consent Address Change
-            "#{PAGE2_KEY}.AuthorizationForRepActClaimantsBehalf[0]": data.consent_address_change == true ? 1 : 0,
+            "#{PAGE2_KEY}.AuthorizationForRepActClaimantsBehalf[0]": data.consent_address_change == true ? 1 : 0
             # Condtions of Appointment
-            "#{PAGE3_KEY}.LIMITATIONS[0]": data.conditions_of_appointment.join(', ')
+            # This is commented out because this field isn't included in the request payload.
+            # "#{PAGE3_KEY}.LIMITATIONS[0]": data.conditions_of_appointment.join(', ')
```

Comment:
might want to double check with @cosu419 what the plans are for this one for the FE. If you didn't want to comment it out, you could nil safe it. `"#{PAGE3_KEY}.LIMITATIONS[0]": data.conditions_of_appointment&.join(', ')`

Reply:
He doesn't need `conditions_of_appointment `so I removed it from everywhere it showed up.